.github/.yamllint.yml

```diff
@@ -1,6 +1,8 @@
 extends: default
 
 rules:
+  comments:
+    min-spaces-from-content: 1
```

Comment:
I chose to add this because my editor was removing the [double space the default calls for](https://yamllint.readthedocs.io/en/stable/rules.html#id6) when saving.